src/View/include/team.html.twig

```diff
@@ -0,0 +1,31 @@
+ <section id="team" class="container-fluid">
+    <div class="col-md-offset-2 col-md-8">
+        <div class="col-xs-12">
+            <h2>L'équipe</h2>
+        </div>
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="row center-team">
+                    <div class="col-xs-3 img-container">
+                            <img class="img-circle" src="{{ TeamMate[0].picture }}" alt="{{ TeamMate[0].alt }}">
```

Comment:
tu as deux fois la même chose, pour les deux membres de l'équipe. Il faudrait plutot faire une boucle for dans le twig